src/main/java/org/openrewrite/java/template/processor/RefasterTemplateProcessor.java

```diff
@@ -340,6 +344,7 @@ public void visitClassDef(JCTree.JCClassDecl classDecl) {
                             if (outerClassRequired) {
                                 out.write("/**\n * OpenRewrite recipes created for Refaster template {@code " + inputOuterFQN + "}.\n */\n");
                                 String outerClassName = className.substring(className.lastIndexOf('.') + 1);
+                                out.write("@javax.annotation.Generated(\"org.openrewrite.java.template.processor.RefasterTemplateProcessor\")\n");
```

Comment:
When added here it's technically not necessary to _also_ add it to each nested class, but that then requires switching around a bunch of code for a harmless annotation. Figured point it out but don't feel like taking that on, hope you all agree.

Reply:
In a similar vein I didn't feel like handling imports, so it's a one liner here.